packages/core/src/components/menu/_common.scss

```diff
@@ -14,9 +14,14 @@ $menu-item-padding-large: ($pt-button-height-large - $pt-icon-size-large) / 2 !d
 $menu-background-color: $white !default;
 $dark-menu-background-color: $dark-gray4 !default;
 
+$menu-item-color-hover: $minimal-button-background-color-hover !default;
+$menu-item-color-active: $minimal-button-background-color-active !default;
+$dark-menu-item-color-hover: $dark-minimal-button-background-color-hover !default;
+$dark-menu-item-color-active: $dark-minimal-button-background-color-active !default;
+
 // customize modifier classes with params.
 // setting modifier to "" will generally apply it as default styles due to & selectors
-@mixin menu-item($disabled-selector: ".pt-disabled", $hover-selector: ":hover") {
+@mixin menu-item($active-selector: ".pt-active", $disabled-selector: ".pt-disabled") {
```

Comment:
I'm not sure I see the value in having these as mixin parameters. If the active selector changes, it will change for the entire system, not just here. We don't seem to be doing that in other places where we use `.pt-active` and `.pt-disabled`